src/app/components/forms/game/build/build.ts

```diff
@@ -107,6 +107,40 @@ export default class FormGameBuild extends BaseForm<GameBuildFormModel>
 		}/${this.model!.id}`;
 	}
 
+	get pollUrl() {
+		return `/web/dash/developer/games/builds/poll-progress/${this.game.id}/${this.package.id}/${
+			this.release.id
+		}/${this.model!.id}`;
+	}
+
+	get shouldPollProgress() {
+		return this.model && this.model.status === GameBuild.STATUS_ADDING && !this.archiveError;
+	}
+
+	get archiveError() {
+		if (!this.model) {
+			return '';
+		}
+
+		if (this.model.hasError(GameBuild.ERROR_INVALID_ARCHIVE)) {
+			return this.$gettext(
+				`The archive you uploaded looks corrupt, we can't extract it on our end.`
```

Comment:
Would be weird if it was corrupt. I think you meant "corrupted".

Reply:
your face looks corrupted